components/identity/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/OAuth2Service.java

```diff
@@ -56,6 +55,8 @@
 
     private static Log log = LogFactory.getLog(OAuth2Service.class);
 
+    public static final String BEARER_TOKEN_TYPE = "bearer";

```

Comment:
Cannot add public constants here. They need to be added to relevant Constant class. In this case IIRC there should be a constant already defined for "bearer". Please check other places in oauth and oauth.endpoint components.